lib/linked_list.rb

```diff
@@ -7,39 +7,76 @@ def initialize
     @head = nil
   end
 
-  # Time complexity - ?
-  # Space complexity - ?
+  # Time complexity O(1)
+  # Space complexity O(1)
   def add_first(data)
-
+    @head = Node.new(data, @head)
   end
 
-  # Time complexity - ?
-  # Space complexity - ?
+  # Time complexity - O(1)
+  # Space complexity - O(1)
   def get_first
-
+    if !(@head.nil?)
+      return @head.data
+    end
+    @head
```

Comment:
It would be more readable to return nil

```suggestion
    return nil
```